spec/regression/versionMapping.spec.ts

```diff
@@ -20,7 +20,7 @@ test("optimize docs cross-link to main docs", async ({ page }) => {
     "/optimize/apis-tools/optimize-api/optimize-api-authentication/"
   );
 
-  await expect(page).toHaveTitle(/Authorization \| Camunda 8 Docs/);
+  await expect(page).toHaveTitle(/Authentication \| Camunda 8 Docs/);
```

Comment:
Good spot! That title did change!

Reply:
You have no idea how long I stared at that failing test trying to pick out the change 😵‍💫